pyproject.toml

```diff
@@ -1,6 +1,6 @@
 [tool.poetry]
 name = "torch_points3d"
-version = "1.4.6+cu116" # This will be overriden by the CI at publish time
+version = "1.4.5+cu117" # This will be overriden by the CI at publish time
```

Comment:
Should this be 1.4.6+cu1117?  I checked pypi but only saw that wheel for 1.4.5, how come we're going backwards? :rofl:

Reply:
Sorry, it is 1.4.5+cu117, and already published.

Since, we didn't have to juggle with separate builds for supposedly the same version, I thought it's a good idea to point to the actual torch-points3d version (from the public repo) here.

Wdyt?

Reply:
Sure either way works; probably makes sense to delete the newer 'bug patch' version then + just go by the different torch / cuda build numbers you're attaching to the wheel

Reply:
Exactly, I want to drop all the misleading / unintuitive stuff when this is merged + we have a stable state on IOS `main`.

I was scared to do this before after messing up the ongoing OSS release, do you think it's safe now?

Reply:
You're intending to do cuda 12 next right?  
It might be best to drop all misleading wheels after investigating if we can upgrade there or not.

Reply:
I don't think many folks are checking out the status on pypi or altering these versions, just trusting what is there tbh.

Reply:
I was just thinking of keeping the most up-to-date, properly named wheels and drop all the rest. But it's a good idea to wait for CUDA 12 upgrade first 👍